addons/kommander/1.1.x/kommander.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-20"
```

Comment:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-1"
```

Reply:
:heavy_check_mark: the revision should be `${CURRENT_APP_VERSION}-${REV}`, sot his suggestion is :+1: